app/src/main/java/com/simplecity/amp_library/utils/DialogUtils.java

```diff
@@ -0,0 +1,86 @@
+package com.simplecity.amp_library.utils;
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.content.Context;
+import android.graphics.Color;
+import android.support.design.widget.BaseTransientBottomBar;
+import android.support.design.widget.Snackbar;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.NumberPicker;
+import android.widget.TextView;
+
+import com.afollestad.materialdialogs.MaterialDialog;
+import com.simplecity.amp_library.R;
+import com.simplecity.amp_library.ShuttleApplication;
+
+public class DialogUtils {
```

Comment:
Same - needs license header with original source link and class comment description